src/argilla/client/feedback/dataset/mixins.py

```diff
@@ -379,3 +382,380 @@ def unify_responses(
+
+        Args:
+            use_markdown: Set this parameter to True if you want to use markdown in your dataset
+
+        Returns:
+            A `FeedbackDataset` object for preference containing "prompt", "option1" and "option2" fields and a LabelQuestion named "preference"
+        """
+        return cls(
+            fields=[
+                TextField(name="context", use_markdown=use_markdown),
+                TextField(name="prompt", use_markdown=use_markdown),
+                TextField(name="response1", title="Response 1", use_markdown=use_markdown),
+                TextField(name="response2", title="Response 2", use_markdown=use_markdown),
+            ],
+            questions=[
+                LabelQuestion(
+                    name="preference", labels=["Response 1", "Response 2"], description="Choose your preference."
+                )
+            ],
+            guidelines="This is a preference dataset that contains contexts and options. Please choose the option that you would prefer in the given context.",
+        )
+
+    @classmethod
+    def for_natural_language_inference(
+        cls: Type["FeedbackDataset"],
+        labels: List[str] = ["entailment", "neutral", "contradiction"],
+        use_markdown: bool = False,
+    ) -> "FeedbackDataset":
+        """
+        You can use this method to create a basic dataset for natural language inference tasks.
+
+        Args:
+            labels: A list of labels for your dataset
+            use_markdown: Set this parameter to True if you want to use markdown in your dataset
+
+        Returns:
+            A `FeedbackDataset` object for natural language inference containing "premise" and "hypothesis" fields and a LabelQuestion named "label"
+        """
+        return cls(
+            fields=[
+                TextField(name="premise", use_markdown=use_markdown),
+                TextField(name="hypothesis", use_markdown=use_markdown),
+            ],
+            questions=[LabelQuestion(name="label", labels=labels, description="Choose one of the labels.")],
+            guidelines="This is a natural language inference dataset that contains premises and hypotheses. Please choose the label that best describes the relationship between the premise and the hypothesis.",
+        )
+
+    @classmethod
+    def for_proximal_policy_optimization(
+        cls: Type["FeedbackDataset"], context: bool = False, use_markdown: bool = False
+    ) -> "FeedbackDataset":
+        """
+        You can use this method to create a basic dataset for proximal policy optimization tasks.
+
+        Args:
+            use_markdown: Set this parameter to True if you want to use markdown in your dataset
+
+        Returns:
+            A `FeedbackDataset` object for proximal policy optimization containing "context" and "action" fields and a LabelQuestion named "label"
+        """
+        return cls(
+            fields=[
+                TextField(name="context", use_markdown=use_markdown),
```

Comment:
We first want to show context and then prompt after.
Also, `context` 
this should be `TextField(required=False)` because we might want a context field but not always use it